src/assets/lang/locale-en.json

```diff
@@ -9,6 +9,9 @@
 		"selectAll": "All"
 	},
 	"terraFileBrowser": {
+		"error": {
+			"tooLargePayload": "Datei zu groß"
```

Comment:
translation

Reply:
Lint error is fixed, translation @fmutschler